sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiCreateOrUpdateContent .cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ApiManagement.Models
+{
+    /// <summary> Summary of revision metadata. </summary>
+    public partial class ApiCreateOrUpdateContent
+    {
+        /// <summary> A URL to the Terms of Service for the API. MUST be in the format of a URL. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This property has been replaced by TermsOfServiceUriLink", false)]
+        public Uri TermsOfServiceUri { get; set; }
```

Comment:
In customization like this scenario, we are creating a new property to replace an old property, therefore we should make sure both old property and new property could work properly. We need to let them share the same backing field, something like this:
```suggestion
        public Uri TermsOfServiceUri
        {
        	get
        	{
        		return Uri.TryParse(TermsOfServiceLink, out var uri) ? uri : null;
        	}
         	set
         	{
         		TermsOfServiceLink = value.AbsoluteUri;
         	}
         }
```
When we call `model.TermsOfServiceUri`, we parse from the uri string (the `TermsOfServiceLink` property) to uri, if succeeded, we return that uri, otherwise return `null`.
When we set this property, such as `model.TermsOfServiceUri = new Uri("something");`, we are actually setting the string property instead.
In this way, from the user's perspective, this property works just fine, they get a Uri when the value in response is a `Uri`, and they could set value to this property.
The serialization and deserialization code should change automatically by the generator to use the string property, therefore there should never be a data loss.

Reply:
we should do similar thing to all other Uri properties.